doc/classes/EditorSettings.xml

```diff
@@ -238,6 +238,9 @@
 		<member name="docks/scene_tree/center_node_on_reparent" type="bool" setter="" getter="">
 			If [code]true[/code], new node created when reparenting node(s) will be positioned at the average position of the selected node(s).
 		</member>
+		<member name="docks/scene_tree/hide_filtered_out_parents" type="bool" setter="" getter="">
+			If [code]true[/code], the scene tree dock will only show nodes that match the filter, without showing parents that don't.
```

Comment:
This could also mention that you can change the setting in `⋮` menu.

Reply:
How should I refer to it? Just ⋮ menu?

Reply:
Not sure tbh.
Maybe "Scene dock top menu".